submission/proxy_server.go

```diff
@@ -72,15 +73,20 @@ func (s *ProxyServer) HandleAddPreChain(w http.ResponseWriter, r *http.Request)
 		http.Error(w, fmt.Sprintf("proxy: failed to parse add-pre-chain body: %s", err), rc)
```

Comment:
nit: The error message says "add-pre-chain", but this now also handles add-chain, so the error message may be incorrect.

Reply:
Fixed.